package.json

```diff
@@ -18,6 +18,7 @@
     "react-scripts": "4.0.3",
     "redux": "^4.1.2",
     "redux-logger": "^3.0.6",
+    "redux-thunk": "^2.4.0",
```

Comment:
- [ ] Kindly deploy the project online and include the live link in the readme.

Reply:
Hello @mikenath223, I will do it, but can you kindly explain why this is a requirement?

Reply:
Please have in mind that I deployed to GitHub Pages, as referenced in the README.